app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsActionMultisendFragment.kt

```diff
@@ -0,0 +1,89 @@
+package io.gnosis.safe.ui.transactions.details
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.LinearLayout
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import io.gnosis.data.backend.dto.ValueDecodedDto
+import io.gnosis.safe.R
+import io.gnosis.safe.ScreenId
+import io.gnosis.safe.databinding.FragmentTransactionDetailsActionMultisendBinding
+import io.gnosis.safe.di.components.ViewComponent
+import io.gnosis.safe.ui.base.fragment.BaseViewBindingFragment
+import io.gnosis.safe.ui.transactions.details.view.MultisendActionView
+import io.gnosis.safe.utils.BalanceFormatter
+import io.gnosis.safe.utils.ParamSerializer
+import pm.gnosis.model.Solidity
+import pm.gnosis.utils.asEthereumAddressString
+import javax.inject.Inject
+
+class TransactionDetailsActionMultisendFragment : BaseViewBindingFragment<FragmentTransactionDetailsActionMultisendBinding>() {
+
+    override fun screenId() = ScreenId.TRANSACTIONS_DETAILS_ACTION_LIST
+
+    private val navArgs by navArgs<TransactionDetailsActionMultisendFragmentArgs>()
+    private val decodedValues by lazy { paramSerializer.deserializeDecodedValues(navArgs.decodedValues) }
+
+    @Inject
+    lateinit var balanceFormatter: BalanceFormatter
+
+    @Inject
+    lateinit var paramSerializer: ParamSerializer
```

Comment:
I haven't seen what this class looks like, but couldn't we just use `@Parcelize` on `DecodedValue` class?

Reply:
some parameter types are not directly supported by parcelize